src/Fields/interpolate.jl

```diff
@@ -211,6 +211,23 @@ end
     return (ii, jj, kk)
 end
 
+"""
+    truncate_fractional_indices(fi, fj, fk)
+
+Truncate _fractional_ indices output from fractional indices `fi, fj, fk` to integer indices, dealing
+with `nothing` indices for `Flat` domains.
+"""
+@inline function truncate_fractional_indices(fi, fj, fk)
+    i = _truncate_fractional_index(fi)
```

Comment:
```suggestion
    i = truncate_fractional_index(fi)
```

since one function is "indices" and the other is "index" I don't think you need to disambiguiate the function names with a leading underscore.

Reply:
I've changed the name